src/main/java/com/google/devtools/build/lib/skyframe/BazelSkyframeExecutorConstants.java

```diff
@@ -29,6 +29,8 @@ private BazelSkyframeExecutorConstants() {}
   public static final ImmutableSet<PathFragment> HARDCODED_IGNORED_PACKAGE_PREFIXES =
       ImmutableSet.of();
 
+  public static final PathFragment BAZELIGNORE_PATH = PathFragment.create(".bazelignore");
```

Comment:
this constant can be inlined -- no reason for it to be a ctor argument, nor a constant in a separate location from its only usage

Reply:
Actually, it was used in two places, but it could be fixed, so you ended up being right :) Done.